### PR TITLE
RHDEVDOCS-4935 - Logging 5.6.4 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10+

Issue: [RHDEVDOCS-4935](https://issues.redhat.com//browse/RHDEVDOCS-4935)

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.6.4.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. This resulted in LokiStack pods crashlooping when the cluster pod network was configured with a non-private IP network range. With this update, the LokiStack administrator can now choose to use the pod network for the memberlist configuration. This resolves the issue and prevents LokiStack pods from crashlooping when the cluster pod network is configured with a non-private IP network range. (link:https://issues.redhat.com/browse/LOG-3814[LOG-3814])
+
+* Before this update, if the `tls.insecureSkipVerify` field was set to `true`, the Cluster Logging Operator would generate an incorrect configuration. As a result, the Operator would fail to send data to Elasticsearch when attempting to skip certificate validation. With this update, the Cluster Logging Operator generates the correct TLS configuration even when `tls.insecureSkipVerify` is enabled. As a result, data can be sent successfully to Elasticsearch even when attempting to skip certificate validation. (link:https://issues.redhat.com/browse/LOG-3838[LOG-3838])
+
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-rn-5.6.4.adoc:24]: I think we might need to add [LOG-3763](https://issues.redhat.com/browse/LOG-3763)

File: modules/cluster-logging-rn-5.6.4.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. As a result, when the {product-title} cluster pod network was configured with a non-private IP network range, the LokiStack pods would crashloop. With this update, the LokiStack administrator now has the option to use the pod network for the memberlist configuration. This resolves the issue and prevents the LokiStack pods from crashlooping when the {product-title} cluster pod network is configured with a non-private IP network range. (link:https://issues.redhat.com/browse/LOG-3454[LOG-3454])
+
+* Before this update, when the `tls.insecureSkipVerify` option was set to `true`, the Cluster Logging Operator would generate incorrect configuration. As a result, the operator would fail to send data to Elasticsearch when attempting to skip certificate validation. With this update, the Cluster Logging Operator generates the correct TLS configuration even when `tls.insecureSkipVerify` is enabled. As a result, data can be sent successfully to Elasticsearch even when attempting to skip certificate validation. (link:https://issues.redhat.com/browse/LOG-3475[LOG-3475])
+
```
Comment (rh-tokeefe) [inline modules/cluster-logging-rn-5.6.4.adoc:14]: I think the second sentence in the write-up for LOG-3640 reads more clearly if we drop the word "only"

This resulted in some of the received logs including the structured message, while others did not.

File: modules/cluster-logging-rn-5.6.4.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-rn-5.6.4.adoc:26]: Added

File: modules/cluster-logging-rn-5.6.4.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-4_{context}"]
+= Logging 5.6.4
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1311[OpenShift Logging Bug Fix Release 5.6.4].
+
+[id="openshift-logging-5-6-4-bug-fixes"]
+== Bug fixes
+* Before this update, when LokiStack was deployed as the log store, the logs generated by Loki pods were collected and sent to LokiStack. With this update, the logs generated by Loki are excluded from collection and will not be stored. (link:https://issues.redhat.com/browse/LOG-3280[LOG-3280])
+
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. As a result, when the {product-title} cluster pod network was configured with a public IP range, the LokiStack pods would crashloop. With this update, the LokiStack administrator now has the option to use the pod network for the memberlist configuration. This resolves the issue and prevents the LokiStack pods from entering a crashloop state when the {product-title} cluster pod network is configured with a public IP range. (link:https://issues.redhat.com/browse/LOG-3454[LOG-3454])
```
Comment (periklis) [inline modules/cluster-logging-rn-5.6.4.adoc:11]: The release notes text here seems not to match the referenced bug ticket. AFAIU the ticket has no releases notes yet.

File: modules/logging-rn-5.6.4.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+:_content-type: REFERENCE
+[id="logging-release-notes-5-6-4_{context}"]
+= Logging 5.6.4
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1311[OpenShift Logging Bug Fix Release 5.6.4].
+
+[id="logging-5-6-4-bug-fixes"]
+== Bug fixes
+* Before this update, when LokiStack was deployed as the log store, the logs generated by Loki pods were collected and sent to LokiStack. With this update, the logs generated by Loki are excluded from collection and will not be stored. (link:https://issues.redhat.com/browse/LOG-3280[LOG-3280])
+
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. As a result, when the {product-title} cluster pod network was configured with a public IP range, the LokiStack pods would crashloop. With this update, the LokiStack administrator now has the option to use the pod network for the memberlist configuration. This resolves the issue and prevents the LokiStack pods from entering a crashloop state when the {product-title} cluster pod network is configured with a public IP range. (link:https://issues.redhat.com/browse/LOG-3454[LOG-3454])
```
Comment (periklis) [inline modules/logging-rn-5.6.4.adoc:11]: As mentioned above in the `cluster-logging-rn-5-6-4.adoc.

**Review by periklis — CHANGES_REQUESTED** [review]

**Comment by libander** [comment]
@periklis - added RN text for LOG-3454, would you double check me on it please?

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**